### DISCUSSION TITLE
Widget as a Package

### DISCUSSION THREAD

**Discussion by philipnjuguna66** [filamentphp/filament#5827] [Ideas]
Could Statis Widgets be added as a package, similar to Forms, Tables, and Notifications? It would be highly beneficial for developers to have access to such a feature especially on existing projects.

**Comment by ManojKiranA** [comment]
Already proposed in [discussion](https://github.com/filamentphp/filament/discussions/4400) for v3 and it's [implemented](https://github.com/filamentphp/filament/tree/3.x/packages/widgets)
![image](https://user-images.githubusercontent.com/30294553/222357941-44b426c1-e812-4929-b3c6-1515679a0ccf.png)

**Reply by philipnjuguna66** [reply]
great
